operators/src/processing/reprojection.rs

```diff
@@ -453,13 +461,25 @@ where
     ) -> Result<BoxStream<'a, Result<Self::Output>>> {
         // calculate the intersection of input and output srs in both coordinate systems
         // TODO: do this in initialization?
-        let valid_bounds_in = Self::valid_bounds::<SpatialPartition2D>(self.from, self.to)?;
-        let valid_bounds_out = Self::valid_bounds::<SpatialPartition2D>(self.to, self.from)?;
+        let (valid_bounds_in, valid_bounds_out) = if let Some(data_bounds) = self.data_bounds {
```

Comment:
isn't this the same as in line 58?

Reply:
it's the same logic but the other way around. Here, we calculate the resolution for **querying** the source and in line 58 we calculate the maximum **output** resolution.